config/routes.rb

```diff
@@ -164,11 +164,11 @@
     resources :elections, path: :val, only: :index do
       collection do
         resources :nominations, controller: 'elections/nominations',
-          path: :nominera, only: [:create] do
-          get '', action: :new, on: :collection, as: :new
+                                path: :nominera, only: [:create] do
+          get "", action: :new, on: :collection, as: :new
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.